content/tutorials/2.projects/use-directus-templates-in-clay-for-data-enrichment.md

```diff
@@ -0,0 +1,417 @@
+---
+id: 8da0ec5c-4865-41fd-a4fe-073e354af16b
+title: Use Directus with Clay for Data Enrichment
+description: Connect Directus with Clay to automate data enrichment and sync content between platforms using webhooks and HTTP API templates.
+technologies:
+  - clay
+---
+
+Connect your Directus instance with Clay to automate data enrichment, sync content, and build powerful workflows between your content platform and Clay's data enrichment tools.
+
+## Before You Start
+
+### Set Up Your Directus Project
+
+You'll need a Directus project with:
+
+- Collections set up with the data you want to work with
+- Proper permissions configured for the operations you plan to use
+- API access enabled and a valid API token
+
+If you don't already have a Directus project, the easiest way to get started is with our [managed Directus Cloud service](https://directus.cloud).
+
+### Set Up Clay
+
+Make sure you have a Clay account and are familiar with:
+
+- Creating enrichment columns
+- Basic Clay workflow concepts
+- How to search for and use templates
+
+## How to Connect Directus and Clay
+
+There are two separate ways to connect Directus and Clay, each with different setup processes:
```

Comment:
What if we split this tutorial into 2 seperate tutorials since there are 2 ways to use Clay with Directus? This is a lot of content to read through in this tutorial.